src/main/java/com/fincons/taskmanager/service/taskService/impl/TaskServiceImpl.java

```diff
@@ -51,11 +49,11 @@ public Task updateTaskByCode(String taskCode, Task task) {
         List<Task> tasks = taskRepository.findAll();
         Task taskExisting = validateTaskByCode(taskCode);
 
-        List<Task> tasksWithoutTaskCodeChosed = new ArrayList<>();
+        List<Task> tasksWithoutTaskCodeChosen = new ArrayList<>();
```

Comment:
what is this method supposed to do

Reply:
This is a put method

Reply:
tasksExcludingSelectedTask use this name